Core/Core/People/PeopleListEnrollment.swift

```diff
@@ -0,0 +1,82 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2023-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import Foundation
+import CoreData
+
+final public class PeopleListEnrollment: NSManagedObject {
+    @NSManaged public var id: String?
+    @NSManaged public var canvasContextID: String?
+    @NSManaged public var role: String?
+    @NSManaged public var roleID: String?
+    @NSManaged public var stateRaw: String?
+    @NSManaged public var userID: String?
+    @NSManaged public var type: String
+    @NSManaged public var course: Course?
+    @NSManaged public var courseSectionID: String?
+    @NSManaged public var observedUser: PeopleListUser?
```

Comment:
As I see this is the user enrolled in this enrollment so a `user` or `enrolledUser` name would be better. I think about the parent-student relationship when I read `observedUser`.

Reply:
refactored.